docs/ref/forms/api.txt

```diff
@@ -1441,25 +1464,38 @@ Methods of ``BoundField``
         >>> print(bound_form["subject"].value())
         hi
 
+.. _custom-boundfield:
+
 Customizing ``BoundField``
 ==========================
 
-If you need to access some additional information about a form field in a
-template and using a subclass of :class:`~django.forms.Field` isn't
-sufficient, consider also customizing :class:`~django.forms.BoundField`.
+There are multiple ways to use a custom :class:`.BoundField`:
+
+* A project can define a project-wise :class:`.BoundField` class using
+  :attr:`.BaseRenderer.bound_field_class` and :setting:`FORM_RENDERER`.
 
-A custom form field can override ``get_bound_field()``:
+* A form can override :attr:`.Form.bound_field_class` or use the constructor
+  argument.
+
+* A custom form field can override :attr:`.Field.bound_field_class` or use the
+  constructor argument.
+
+* A custom form field can override :meth:`.Field.get_bound_field()`:
 
 .. method:: Field.get_bound_field(form, field_name)
 
     Takes an instance of :class:`~django.forms.Form` and the name of the field.
     The return value will be used when accessing the field in a template. Most
-    likely it will be an instance of a subclass of
-    :class:`~django.forms.BoundField`.
+    likely it will be an instance of a subclass of :class:`.BoundField`.
```

Comment:
Is this diff line necessary?

Reply:
I suggested this change ```:class:`.BoundField`.``` is being used in this section as to me it renders better and is easier to read in the short form